### DISCUSSION TITLE
Upgrading Slack?

### DISCUSSION THREAD

**Discussion by kubu4** [RobertsLab/resources#1316] [General]
Do we have any interest in upgrading Slack to "Pro"? This would allow searching all of our archived messages and allow for unlimited app integration. These are the two advantages for me, personally. 

Cost would be ~$7 member/month. Doesn't sound bad, but now that I'm looking at all the members, it would be costly; ~$300/month!. That pricing, btw, is based off a yearly subscription. If paying month-to-month, the cost goes up to $8 member/month.

**Comment by sr320** [comment]
I think not- I think if there is something we want for posterity (searching), it should live elsewhere. Thinking slack as more ephemeral, though open to hearing other side. For instance, what are you needing to search for?

**Reply by kubu4** [reply]
Okay. Works for me.

Specifically, I'm looking for an `awk` "script" (may have only been two or three lines?) that I wrote. Related to counting something related to Bismark. The post compared my results with the results of a script Giles had provided. My `awk` "script" matched the output from a Bismark tool (which, we didn't realize existed at the time), whereas Giles' was off.

**Reply by sr320** [reply]
Nice case study... 
Curious - all scripts do not live is central place for searching?

And when I am proud of myself for a 2-3 liner I drop in as Show and Tell
<img width="344" alt="Notification_Center" src="https://user-images.githubusercontent.com/4442039/137350314-e8470241-f075-4fcd-b871-a36091f814ea.png">
!

**Reply by kubu4** [reply]
> all scripts do not live is central place for searching?

Well, no. It was just a quick "one off" that I threw together as part of a Slack thread regarding counting CG coverage from Bismark data for something. So, it was never officially a script; particularly because, during that Slack thread, we realized that Bismark could already perform this task - we just hadn't realized it until some exploration triggered by that Slack thread.

So, at that point, there wasn't a need to preserve the `awk` stuff. Now, I'm just wanting it to see how I used it. I'm pretty sure it went through a file, compared a field with the same field on the next line and then incremented a counter. I figured it'd be faster for me to just refer to that Slack thread again instead of searching the internet and reading through a bunch of various stackoverflow discussions to remember how it's done.

**Reply by kubu4** [reply]
Also, the Slack thread preceded the deployment of the "Discussions" feature on GitHub.
